tests/test_client.py

```diff
@@ -937,6 +943,25 @@ def test_get_order_decisions(self):
             assert(response.body['decisions']['payment_abuse']['decision']['id'] == 'decision7')
             assert(response.body['decisions']['promotion_abuse']['decision']['id'] == 'good_order')
 
+    def test_get_session_decisions(self):
+        mock_response = mock.Mock()
+        mock_response.content = '{"decisions":{"account_abuse": {"decision": {"id": "session_decision"},"time": 1461963839151,"webhook_succeeded": true}}}'
```

Comment:
I meant `account_takeover` here because session decision is only available for ATO, though it doesn't affect the test result itself.